package.json

```diff
@@ -23,6 +23,6 @@
     "nan": "^2.14.1"
   },
   "devDependencies": {
-    "tree-sitter-cli": "^0.19.4"
+    "tree-sitter-cli": "^0.19.5"
```

Comment:
can we retain 0.19.4 ? I'm sure anyways node is going to use the latest of the minor version, since `^` is present. But I'm planning to do a upgradation to version 0.20 later, so can we revert this change alone?